docs/test_docs.py

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env python3
+
+import os
+import shutil
+from subprocess import Popen, PIPE
+
+
+def main():
+    if os.path.exists('_build'):
+        shutil.rmtree('_build')
+    p = Popen(['make', 'html'], stderr=PIPE)
+    _, err = p.communicate()
+    err = [i for i in err.decode().splitlines() if 'WARNING' in i]
+    assert len(err) == 0, '\n'.join(err)
+
+
+if __name__ == '__main__':
+    main()
```

Comment:
maybe sphinx has some built-in functions to check this. This file is too brute-force.

Reply:
I think so, but the sphinx makefile only provides "make doctest" which is not what our expected.

Reply:
Simply remove the ``sphinx.ext.imgmath`` extension in conf.py works just fine.